src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -7,18 +7,18 @@
 import net.dv8tion.jda.core.EmbedBuilder;
 import net.dv8tion.jda.core.JDA;
 import net.dv8tion.jda.core.Permission;
-import net.dv8tion.jda.core.entities.Emote;
-import net.dv8tion.jda.core.entities.Guild;
-import net.dv8tion.jda.core.entities.Member;
-import net.dv8tion.jda.core.entities.Message;
-import net.dv8tion.jda.core.entities.Role;
-import net.dv8tion.jda.core.entities.TextChannel;
-import net.dv8tion.jda.core.entities.User;
+import net.dv8tion.jda.core.entities.*;
```

Comment:
:eyes:

Reply:
How many were removed? If it's above 10 then it's fine

Reply:
7 where removed. and i though we only * imported when all of them in the package is used witch is why commands only recently switched over

Reply:
Nah, commands switch over because it was over 10. If it's 7 I'm inclined to leave it cause that is still quite a few. Opinion @WalshyDev pls

Reply:
This wasma change done by William, his numbers are always low. Regardless I think 7 is ok but I'd prefer to not have a wildcard here

Reply:
:eyes: I'll fix it then